### DISCUSSION TITLE
Can I set "Show all optios" as default?

### DISCUSSION THREAD

**Discussion by PeterM-9** [TechbeeAT/jtxBoard#1034] [Q&A] [answered]
Hi all,
I today installed jtxBoard from fDroid to my Android 13 phone and love it from the start. I like the clear design and ease of use. But I cannot find the possibility to have "Show all options" for tasks set as default. Background: I usually add a priority to my tasks and this option is not shown in the edit screen, it is only activated after clicking "show all options". Can I show the priority option for tasks always when I create/edit a task?
Thanks for your input and sorry if I missed that setting during my research.
Peter

**Comment by PeterM-9** [comment, marked as answer]
OK, I need more patience. While working on a new entry, I discovered the settings button at the bottom left, which - I think - does exactly what I am asking for.

**Comment by patrickunterwegs** [comment]
Hey @PeterM-9  🙂
So yes, you answered your question yourself 😉 while the Show all options button would only show all options once for the current entry, you can configure the displayed elements through the settings button on the bottom left 👍🙂
